Something/WebApplication/wwwroot/js/apps/onboarding/Onboarding.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import { IOnboardingProps } from "./types/IOnboardingProps";
+import { NavigationMenu } from "../common/components/NavigationMenu/NavigationMenu";
+import { Wizard } from "./components/Wizard";
+import { Layout } from "antd";
+import { PageFooter } from "../common/components/PageFooter/PageFooter";
+import { getParams } from "../common/GlobalService";
+import { PageTypes } from "../common/PageTypeEnum";
+
+import "./Onboarding.scss";
+
+const { Content } = Layout;
+
+export const Onboarding: React.SFC = () => {
+    const params = getParams(PageTypes.ONBOARDING) as IOnboardingProps;
+    if (!params) { return null; }
```

Comment:
Why do you think this will be null ?

Reply:
I think params should be passed as props here.

Reply:
Component should not have responsibility of getting it's props

Reply:
This is the entry level component, it has to get its props from the param. Null check removed.